packages/clay-nav/src/Vertical.tsx

```diff
@@ -32,7 +32,7 @@ interface IItem extends React.ComponentProps<typeof Nav.Item> {
 	/**
 	 * Value of item.
 	 */
-	label?: string;
+	label?: string | React.ReactElement;
```

Comment:
I think instead of the `ReactElement` type we should add `ReactNode` instead because it covers more use cases, like text and not necessarily React elements, like `label={<div />}`.

Reply:
@julien maybe something to keep an eye out for, this is very related to #4112 and #4117. This is important for us to design a more flexible API for collections in general that should reflect in the TreeView since Vertical Navigation will be deprecated in favor of it.

Reply:
I agree with you, @matuzalemsteles, thanks for the comments.

I'll push the recommended changes, let me know if you need anything else !

Ty